packages/php-wasm/compile/build.js

```diff
@@ -156,10 +159,21 @@ if (!requestedVersion || requestedVersion === 'undefined') {
 	process.exit(1);
 }
 
+const [majorPhpVersion, minorPhpVersion] = getArg('PHP_VERSION', false).split(
+	'.'
+);
+
+let bisonVersion = '3';
+
+if (majorPhpVersion <= 7 && minorPhpVersion <= 3) {
```

Comment:
I'd like to keep that logic in the Dockerfile as it never makes sense to use Bison 2.7 with PHP 8.0 or Bison 3.0 with PHP 7.0. Bison versions are tightly coupled with PHP versions. Using a separate BISON_VERSION argument decouples them and shifts some cognitive burden onto the caller, as in "wait, which version am I supposed to use here?".

Reply:
https://github.com/WordPress/wordpress-playground/pull/871/files#r1432295382